ingest/src/main/java/com/microsoft/azure/kusto/ingest/ResourceManager.java

```diff
@@ -61,13 +60,9 @@ public class ResourceManager implements Closeable, IngestionResourceManager {
     private RequestRetryOptions queueRequestOptions = null;
     private RankedStorageAccountSet storageAccountSet;
     private String identityToken;
-    private IngestionResource<ContainerWithSas> containers;
-    private IngestionResource<TableWithSas> statusTable;
-    private IngestionResource<QueueWithSas> queues;
-    private IngestionResource<QueueWithSas> successfulIngestionsQueues;
-    private IngestionResource<QueueWithSas> failedIngestionsQueues;
+    private IngestionResourceSet ingestionResourceSet;
 
-    ResourceManager(Client client, long defaultRefreshTime, long refreshTimeOnFailure, @Nullable CloseableHttpClient httpClient) {
+    public ResourceManager(Client client, long defaultRefreshTime, long refreshTimeOnFailure, @Nullable CloseableHttpClient httpClient) {
```

Comment:
Why are you making it public? this means we are responsible to keep it not broken

Reply:
for some tests
will revert